src/SmallStrainPlasticity/IsotropicHardening.jl

```diff
@@ -0,0 +1,26 @@
+# Isotropic hardening
+abstract type AbstractIsoHard{T} end
+
+# Voce type of isotropic hardening (exponentially saturating)
+struct Iso_Voce{T} <:AbstractIsoHard{T}
+    Hiso::T     # Initial hardening modulus
+    κ∞::T       # Saturation stress
+end
+Iso_Voce(;Hiso, κ∞) = Iso_Voce(Hiso, κ∞)    # Keyword argument constructor
+
+function IsotropicHardening(param::Iso_Voce, λ::Number)
+    param.κ∞ * (1.0 - exp(-param.Hiso * λ / param.κ∞))
```

Comment:
It seems like this function is returning a scalar value, but the the capital "I" in the function name makes it look like constructor of some sort. Maybe call it calculate_hardening(..) or something?

Reply:
Makes sense! And of course we don't need different function names for kinematic / isotropic hardening - still getting used to the power of multiple dispatch:)

Reply:
Also `Iso_Voce` should be `IsoVoce` or `IsotropicVoce` according to Julia naming conventions.

Reply:
Right! I updated to IsotropicHardeningVoce etc. on Friday - I hope this is ok although a bit verbose...
It doesn't seem to have been updated here, I guess it will next time I push as my fork was still private...?